conf.json

```diff
@@ -5,6 +5,7 @@
       "name": "sample_name",
       "disabled": false,
       "connectionType": "kafka_http",
+      "metric_port": 9999,
```

Comment:
we can keep this outside `dmuxItems` but in parallel with `name, dmuxItems & logging`